pkg/oceanbase/operation/restore.go

```diff
@@ -31,9 +31,9 @@ func (m *OceanbaseOperationManager) SetRestorePassword(password string) error {
 	return nil
 }
 
-func (m *OceanbaseOperationManager) StartRestoreWithLimit(tenantName, uri, limitKey, restoreOption string, limitValue any) error {
+func (m *OceanbaseOperationManager) StartRestoreWithLimit(tenantName, uri, restoreOption string, limitKey, limitValue any) error {
 	sqlStatement := fmt.Sprintf(sql.StartRestoreWithLimit, tenantName, limitKey)
-	err := m.ExecWithDefaultTimeout(sqlStatement, uri, limitValue, restoreOption)
+	err := m.ExecWithTimeout(600*time.Second, sqlStatement, uri, limitValue, restoreOption)
```

Comment:
suggest to define timeout for certain scenario in one place to avoid magic number

Reply:
OK, I'll define a constant in another package and reference it.